app/routes/index.js

```diff
@@ -3,10 +3,9 @@ import Route from '@ember/routing/route';
 export default Route.extend({
 
   model() {
-    fetch('https://api.github.com/search/repositories?q=user:ember-learn+NOT+builds+NOT+statusboard+help-wanted-issues:%3E0+archived:false')
+    fetch('/github-repositories')
```

Comment:
This doesn't actually work in the deployed test application because it needs the same configuration for a host as you see in the application adapter https://github.com/ember-learn/ember-help-wanted/blob/master/app/adapters/application.js#L7

this works locally when developing because it has a proxy enabled (for ease of development) but would not work in production until we update the host thing

Reply:
Gotcha. That makes sense relative to the deployed test app now that you mention it.

I could be wrong, but I think there is a proxy enabled for prod, so it should work there? 
https://github.com/jaredgalanis/ember-help-wanted/blob/f81ceb9e3f9c8d25b24dfea45d2e67f522a65b6c/server/proxies/api-proxy.js#L23

Would an acceptable solution be setting an additional env var for deployed test apps? or is that not something that's possible with our deployment strategy?

Reply:
So that proxy is 100% definitely only being used in development, it doesn't actually run in production. 

You can see this in action when you go to https://help-wanted.emberjs.com/learning because the XHR request actually goes to https://ember-help-wanted-server.herokuapp.com/github-issues?group=learning which is configured in the environment https://github.com/ember-learn/ember-help-wanted/blob/master/config/environment.js#L53

The link I already sent you is how to check if it's set https://github.com/ember-learn/ember-help-wanted/blob/master/app/adapters/application.js#L7 essentially you need to prefix the fetch url with the host if it is set 👍